src/include/cell.i

```diff
@@ -154,9 +154,15 @@ __cell_pack_addr_validity(WT_SESSION_IMPL *session, uint8_t **pp, wt_timestamp_t
         ++*pp;
 
         flags = 0;
+        if (prepare)
+            LF_SET(WT_CELL_PREPARE);
+        if (oldest_durable_ts != WT_TS_NONE) {
+            WT_IGNORE_RET(__wt_vpack_uint(pp, 0, oldest_durable_ts));
+            LF_SET(WT_CELL_TS_DURABLE_START);
+        }
         if (newest_durable_ts != WT_TS_NONE) {
```

Comment:
comparison to be with WT_TS_MAX as initialization is with default value as WT_TS_MAX.

Reply:
Fixed.